data.sql

```diff
@@ -4,3 +4,9 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, true, 8.00);
```

Comment:
- [ ] kindly consider adding Agumon's data well as instructed to avoid future errors re-check the below requirement
- > Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.

consider re-checking all data and out put of your querries as well